### PR TITLE
Edits to new graphqlws subscription docs

### PR DESCRIPTION
**Pull request by StephenBarlow**
Just some minor cleanup

### CODE REVIEW

File: docs/source/api/link/apollo-link-ws.md [hunk 1]
```diff
@@ -15,9 +15,6 @@ The `WebSocketLink` is a [terminating link](./introduction/#the-terminating-link
 ```shell
 npm install subscriptions-transport-ws
 ```
-
-> **Note**: The `subscriptions-transport-ws` library is not actively maintained. We recommend the use of the `graphql-ws` library instead. These libraries layer different protocols on top of WebSockets, so you do need to ensure you are using the same library in your server and any clients that you support. To use `graphql-ws` from Apollo Client, use the [`GraphQLWsLink` link from `@apollo/client/link/subscriptions](./apollo-link-subscriptions) instead.
```
Comment (StephenBarlow) [inline docs/source/api/link/apollo-link-ws.md:19]: This felt largely redundant with the notice at the top, but let me know if anything is lost with cutting this one

Comment (glasser) [reply docs/source/api/link/apollo-link-ws.md:19]: Oops, not sure how this happened. Is it worth keeping (up top) an acknowledgement that you need to change both server and client?

File: docs/source/data/subscriptions.mdx [hunk 2]
```diff
@@ -22,13 +22,13 @@ You _should_ use subscriptions for the following:
...
 
-> **Note**: When looking at the source code of an implementation to determine which protocol it supports, you will find that the libraries uses different strings as the "WebSocket subprotocol". Confusingly, `subscriptions-transport-ws` uses the `graphql-ws` subprotocol and `graphql-ws` uses the `graphql-transport-ws` subprotocol! In these docs, when we say "`graphql-ws`" we are referring to the _library_ `graphql-ws`, not the subprotocol `graphql-ws`, which is the other project.
+> **Note**: Confusingly, the `subscriptions-transport-ws` library calls its _WebSocket subprotocol_ `graphql-ws`, and the `graphql-ws` _library_ calls its  subprotocol `graphql-transport-ws`! In this article and throughout the Apollo docs, we refer to the two _libraries_ (`subscriptions-transport-ws` and `graphql-ws`), _not_ the two subprotocols.
```
Comment (glasser) [inline docs/source/data/subscriptions.mdx:31]: FWIW I'm not sure that this is entirely consistent throughout "the Apollo docs" as Apollo iOS chose a protocol-based naming. I think [their section](https://www.apollographql.com/docs/ios/subscriptions/#graphql-over-websocket-protocols) is pretty clear though.  Just worth knowing I think

**Review by glasser — APPROVED** [review]

**Review by rkoron007 — APPROVED** [review]
Looks great to me!

**Pull request merged**